docs/snippets/angular/button-group-story.ts.mdx

```diff
@@ -14,18 +14,18 @@ export default {
   decorators: [
     moduleMetadata({
       declarations: [Button],
-      imports: [CommonModule],
-    }),
+      imports: [CommonModule]
+    })
```

Comment:
```suggestion
      imports: [CommonModule],
    }),
```

Should these still be here?

Reply:
I'm good either way with this one and the one below. Adjusted them accordingly with the latest commit. And we can keep some consistency in terms of code examples.

Reply:
:aa:  personally,  I really prefer with `,` at the end 🙈  but as you wish